docker/main/scripts/main.py

```diff
@@ -0,0 +1,823 @@
+                )
+                return False
+
+            # get all ttl files that have been processed
+            processed_objects = [] 
+            for filepath in Path(results_path).glob('**/*.ttl'):
+                processed_objects.append(filepath.name)
+
+            # find any missing files that should exist in S3 bucket
+            missing_objects = set(sqs_objects) - set(processed_objects)
+
+            if len(missing_objects) > 0:
+                logging.error("The following %s files were missing from validation results: %s",
+                        str(len(missing_objects)), missing_objects
+                    )
+
+                self._create_verification_output(
+                    results_path,
+                    verification_log,
+                    "The following {0} files were missing from validation results: {1}".format( 
+                        str(len(missing_objects)), missing_objects
+                    )
+                )
+                return False
+            else:
+                logging.info("Successfully verified all  %s files placed on SQS were accounted for in validation results",
+                        str(len(sqs_objects))
+                    )
+
+                self._create_verification_output(
+                    results_path,
+                    verification_log,
+                    "Successfully verified all {0} files placed on SQS were accounted for in validation results".format(
+                        str(len(sqs_objects))
+                    )
+                )
+                return True 
+        else:
+            logging.error("Source log file %s does not exist, unable to verify source files", source_log_path)
+
+            self._create_verification_output(
+                results_path, 
+                verification_log,
+                "Source log file {0} does not exist, unable to verify source files".format(source_log_path)
+            )
+            return False
+
+
+    def _create_verification_output(self, results_path, source_verfication_path, message):
+        """Generates verification file with verification reuslts to be added to final 
+        archive and sets verification message for final report.
+
+        :param str results_path: The local path of the sync'd s3 bucket
+        :param str source_verification_path: The path to place this verification output file
+        :param str message: The message that will be added to the file with the verification 
+            results
+        :raises Exception: Excpetion occured when attempting to write to file 
+        """
+        file_path = '/'.join([results_path, source_verfication_path])
+        try:
+            with open(file_path, "w") as f:
+                print(message, file=f)
+                self.verification = message #set for final report
+        except:
+            logging.error("Error when writing source verification file %s", file_path)
+
```

Comment:
Typo Line 601: reuslts should be results.
Typo Line 600 and 610: You might want to change source_verfication_path to source_verification_path.

Reply:
Not sure I understand your second comment. Is there a typo in there?

Reply:
The variable name source_verfication_path has verification spelled incorrectly.  It isn't necessarily important and the variable when referenced is also spelled incorrectly, so it doesn't effect any code.  I just felt that I should mention it.

Reply:
ah okay, thanks!